openshift.spec

```diff
@@ -149,44 +223,66 @@ do
   echo "+++ INSTALLING ${bin}"
   install -p -m 755 _build/bin/${bin} %{buildroot}%{_bindir}/${bin}
 done
+
+# openshift == atomic-enterprise
+install -p -m 755 _build/bin/openshift _build/bin/atomic-enterprise
+install -p -m 755 _build/bin/atomic-enterprise %{buildroot}%{_bindir}/atomic-enterprise
+
 # Install 'openshift' as client executable for windows and mac
-install -p -m 755 _build/bin/openshift %{buildroot}%{_datadir}/%{name}/linux/oc
-install -p -m 755 _build/bin/darwin_amd64/openshift %{buildroot}%{_datadir}/%{name}/macosx/oc
-install -p -m 755 _build/bin/windows_386/openshift.exe %{buildroot}%{_datadir}/%{name}/windows/oc.exe
+for pkgname in openshift atomic-enterprise
+do
+  install -d %{buildroot}%{_datadir}/${pkgname}/{linux,macosx,windows}
+  install -p -m 755 _build/bin/openshift %{buildroot}%{_datadir}/${pkgname}/linux/oc
+  install -p -m 755 _build/bin/darwin_amd64/openshift %{buildroot}%{_datadir}/${pkgname}/macosx/oc
+  install -p -m 755 _build/bin/windows_386/openshift.exe %{buildroot}%{_datadir}/${pkgname}/windows/oc.exe
+done
+
 #Install openshift pod
 install -p -m 755 images/pod/pod %{buildroot}%{_bindir}/
 
-install -d -m 0755 %{buildroot}/etc/%{name}/{master,node}
 install -d -m 0755 %{buildroot}%{_unitdir}
-install -m 0644 -t %{buildroot}%{_unitdir} rel-eng/openshift-master.service
-install -m 0644 -t %{buildroot}%{_unitdir} rel-eng/openshift-node.service
 
 mkdir -p %{buildroot}%{_sysconfdir}/sysconfig
-install -m 0644 rel-eng/openshift-master.sysconfig %{buildroot}%{_sysconfdir}/sysconfig/openshift-master
-install -m 0644 rel-eng/openshift-node.sysconfig %{buildroot}%{_sysconfdir}/sysconfig/openshift-node
 
-mkdir -p %{buildroot}%{_sharedstatedir}/%{name}
+ln -s %{_bindir}/%{name} %{buildroot}%{_bindir}/oc
+ln -s %{_bindir}/%{name} %{buildroot}%{_bindir}/oadm
```

Comment:
It was decided to use different binary names for openshift and ae products in order for daemon/tools to know, which product is  being used. There are few proposed names for AE binaries in #3953.